main.go

```diff
@@ -11,27 +15,41 @@ import (
 	. "github.com/env0/terratag/terraform"
 	. "github.com/env0/terratag/tfschema"
 	"github.com/env0/terratag/utils"
+
+	"github.com/hashicorp/go-hclog"
 	"github.com/hashicorp/hcl/v2/hclwrite"
-	"log"
-	"strings"
+	"github.com/hashicorp/logutils"
 )
 
 func main() {
-	Terratag()
-}
+	args, isMissingArg := InitArgs()
+	if isMissingArg {
+		return
+	}
+	filter := &logutils.LevelFilter{
+		Levels:   []logutils.LogLevel{"DEBUG", "TRACE", "INFO", "WARN", "ERROR"},
+		MinLevel: logutils.LogLevel("INFO"),
+		Writer:   os.Stderr,
+	}
+	if args.Verbose {
+		filter.MinLevel = logutils.LogLevel("DEBUG")
+	}
+	log.SetOutput(filter)
```

Comment:
Perhaps we should extract these to a named function -  
1. It'd give a header to what's being done
2. Perhaps that function would be where we set the log level for tfschema

Reply:
+1. will do post merge, since #62 continues where this pull request ends.